### DISCUSSION TITLE
C# simple coding for control flow

### DISCUSSION THREAD

**Discussion by ukepogluradr** [elsa-workflows/elsa-core#1450] [Q&A] [answered]
hi,

I am new on workflow based app development.  Could you write a simple code snippet about usage of control flow (For,While,Finish,If ...)? I did't find any document about this.

thanks in advance,
umit

**Comment by sfmskywalker** [comment, marked as answer]
Hi @ukepogluradr , take a look at the samples folder. There’s a sub folder called “console” that has a bunch of simple examples, including  ones demonstrating For and While activities.

**Reply by ukepogluradr** [reply]
thank you for your replay
